program/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "mpl-rooster"
+name = "rooster"
```

Comment:
Was this intentional?

Reply:
Yes. Was considering publishing it and `rooster` is already taken, but right now Token Metadata tests point to `rooster` so if we decide to publish it we can change this later.